File: codingTest/baekjoon/javascript/11650.js
var rl = require("readline").createInterface({ input: process.stdin, output: process.stdout });
var list = [];
rl.on("line", line => {
  list.push(line.split(" "));
}).on("close", () => {
  list.shift();
  list.sort((a, b) => a[0] - b[0] || a[1] - b[1]);
  var result = "";
  list.forEach(e => (result += e.join(" ") + "\n"));
  console.log(result);
});

File: codingTest/baekjoon/python/11650.py
n = int(input())
arr = []
for _ in range(n):
    x, y = map(int, input().split())
    arr.append((x, y))

result = sorted(arr)
s = ""
for i in result:
    s += "%d %d\n" % (i[0], i[1])
print(s)